Tests/XMTPTests/ClientTests.swift

```diff
@@ -20,6 +20,38 @@ class ClientTests: XCTestCase {
 		_ = try await Client.create(account: fakeWallet)
 	}
 
+	func testPassingMLSEncryptionKey() async throws {
+		try TestConfig.skipIfNotRunningLocalNodeTests()
+
+		let bo = try PrivateKey.generate()
+		let key = try Crypto.secureRandomBytes(count: 32)
+
+		_ = try await Client.create(
+			account: bo,
+			options: .init(
+				api: .init(env: .local, isSecure: false),
+				enableAlphaMLS: true,
+				mlsEncryptionKey: key
+			)
+		)
+
+		do {
+			_ = try await Client.create(
+				account: bo,
+				options: .init(
+					api: .init(env: .local, isSecure: false),
+					enableAlphaMLS: true,
+					mlsEncryptionKey: nil // No key should error
+				)
+			)
+
+			XCTFail("did not throw")
+		} catch {
+			XCTAssert(true)
+		}
+	}
+
+
```

Comment:
Might be nice to add a test for the creating with a bundle scenario since it's a bit of an outlier.